File: rest/message/list-get-example-2/list-get-example-2.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Get all messages that match the given values
@client.account.messages.list(
  to: 'to_number',
  from: 'from_number',
  date_sent: Time.new('2015-04-01T00:00:00Z')
).each do |message|
  # print the body for each message
  puts message.body
end

File: rest/short-codes/list-get-example-2/list-get-example-2.5.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

$shortCodes = $client->shortCodes->read(
    array("shortCode" => "67898")
);

// Loop over the list of shortCodes and echo a property for each one
foreach ($shortCodes as $shortCode) {
    echo $shortCode->smsMethod;
}
